### PR TITLE
doc/rados: edit ceph-conf.rst

### PR DESCRIPTION
**Pull request by zdover23**
Edit doc/rados/configuration/ceph-conf.rst.

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [x] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [x] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

File: doc/rados/configuration/ceph-conf.rst [hunk 1]
```diff
@@ -4,91 +4,93 @@
...
-exception to this: the :confval:`mon_host` option that identifies the addresses
-of the cluster's monitors.  When DNS is used to identify monitors, a local Ceph
-configuration file can be avoided entirely.
+- :confval:`mon_data`, :confval:`osd_data`, :confval:`mds_data`, 
+  :confval:`mgr_data`, and similar options that define which local directory
+  the daemon stores its data in.
+- :confval:`keyring`, :confval:`keyfile`, and/or :confval:`key`, which can be 
+  used to specify the authentication credential to use to authenticate with the
+  monitor. Note that in most cases the default keyring location is in the data
+  directory specified above.
```
Comment (zdover23) [inline doc/rados/configuration/ceph-conf.rst:86]: Which directory is specified above?

Comment (anthonyeleven) [reply doc/rados/configuration/ceph-conf.rst:86]: I think this refers to the metadata directory.

Comment (anthonyeleven) [reply doc/rados/configuration/ceph-conf.rst:86]: But isn't this in `/etc/ceph`?

**Review by anthonyeleven — APPROVED** [review]

File: doc/rados/configuration/ceph-conf.rst [hunk 2]
```diff
@@ -4,91 +4,93 @@
...
-configuration options.  Be careful to understand the consequences before
-overriding default values, as it is possible to significantly degrade the
-performance and stability of your cluster.  Also note that default values
-sometimes change between releases, so it is best to review the version of
-this documentation that aligns with your Ceph release.
+least one :term:`Ceph Metadata Server` (``ceph-mds``). Clusters that support
+:term:`Ceph Object Storage` run Ceph RADOS Gateway daemons (``radosgw``).
+
+Each daemon has a number of configuration options, and each of these
+configuration options has a default value. You may adjust the behavior of the
```
[the remark this answers is not in the document]
Comment (zdover23) [reply doc/rados/configuration/ceph-conf.rst:20]: Accepted with minor alteration.

**Comment by zdover23** [comment]
https://github.com/ceph/ceph/pull/52448 - Reef backport
https://github.com/ceph/ceph/pull/52449 - Quincy backport

**Pull request merged**